examples/abbas_work_area/datastore.py

```diff
@@ -0,0 +1,7 @@
+from abc import ABC, abstractmethod
+
+
+class DataStore(ABC):
+    @abstractmethod
+    def format(self, contact_record):
```

Comment:
I think it would be better to add multiline comments to tell the devs, what this format is supposed to do